law-tayh-3al-mariiii55/main.cpp

```diff
@@ -0,0 +1,19 @@
+#include<iostream>
+#include "MinHeap.h"
+using namespace std;
+int main()
+{
```

Comment:
Please clean the main file after you test your work, it will not conflict with us when we need to test our work.